### ISSUE TITLE
Change hiccup compact syntax for classes to a more idiomatic one

### ISSUE THREAD

**Issue by Heliosmaster** [larrychristensen/orcpub#24]
Throughout the project, the hiccup syntax for classes used is the short-hand version

```
[:div.example-class]
```

I suggest to change it to the more idiomatic

```
[:div {:class "example-class}]
```
This is not only a cosmetic change, but it also helps with some IDE (cursive, for example).

I understand that this might not be something that needs to be done all at once, but it might be good to discuss whether a "style guide" is needed, in order to make the code more uniform.

Thoughts?

**Comment by codeGlaze** [comment]
If it changes how orcbrews are imported/exported ... This is going to fubar a lot of content.

**Comment by Heliosmaster** [comment]
@codeGlaze why does it change how orcbrews are imported and imported?

**Comment by larrychristensen** [comment]
@Heliosmaster I'm not sure it's more idomatic, but I'm open to being convinced. I do find the hiccup much less verbose, so I typically only use the {:class "example-class"} syntax when my classes are dynamically set. If it provides a lot of value to Cursive users, however, I could be convinced to make the switch. Could you elaborate on how it helps in Cursive?

**Comment by Heliosmaster** [comment]
@codeGlaze yes it definitely makes sense to keep track also of that. As the two syntaxes are compatible i wouldn't worry about the generated files. When we export again they would be different but still valid. I think it's only relevant for the code here.

@larrychristensen it helps a little bit because cursive treats each as a separate expression. `[:div]` and `[:div.class]` are considered two different expressions `:div` and `:div.class` respectively. So it doesn't play super nicely with the code analysis features. And, of course, `:div.class1.class2` is different from `:div.class2.class1` but you'd get that anyway using the `[:div {:class "class1 class2"}]` style, i guess).

But to be honest, it's something that at this point is just a nit to pick and not useful. I opened the issue because it was literally the first thing i noticed opening the code and as at work we decided on the aforementioned convention I thought it'd be nice to mention it too. I'm ok for closing this :)

**Comment by dhleong** [comment]
@Heliosmaster Do you have a specific example of how the more verbose syntax works better with Cursive? I don't use Cursive, but like @larrychristensen I've only ever used that syntax when necessary.
